modules/server/src/test/ts/ClisTest.ts

```diff
@@ -60,8 +60,9 @@ describe('Clis.forAuto', () => {
       useSelenium: false,
       extraBrowserCapabilities: '',
       skipResetMousePosition: false,
-      webdriverPort: 4444,
-      wipeBrowserCache: false
+      wipeBrowserCache: false,
+      remote: '',
```

Comment:
Should there be a test for the remote options?

Reply:
yes, but there's not enough dev time left in the schedule